src/modules/broadcasting/repositories/in-memory/InMemoryMessagesRepository.ts

```diff
@@ -52,7 +52,7 @@ export class InMemoryMessagesRepository implements IMessagesRepository {
 
     if (query) {
       messageList = this.items.filter(message =>
-        message.subject.value.includes(query)
+        message.subject.value.includes(query.toLowerCase())
```

Comment:
Shoudn't we also lowercase the `message.subject.value` as well before comparing to query? As we are only lowercasing the query, if the subject contain capitalized chars, it won't be found.